### PR TITLE
Theme demo

### PR DESCRIPTION
**Pull request by markwylde**
Not ready yet, but any ideas how we can do animations on a details/summary widget. Seems to be trickier than I would have liked.

Resolves: https://github.com/votemike/react-accordion/issues/18

### CODE REVIEW

**Comment by votemike** [comment]
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details

> Unfortunately, at this time there's no built-in way to animate the transition between open and closed.
😿

**Comment by votemike** [comment]
Seeing as it's just a demo of what can be done and not in-built.
Want to do it a more complicated none CSS way?

**Comment by votemike** [comment]
For the moment, as the example, perhaps just copy the "Skills and Expertise" section here: https://www.checkatrade.com/trades/jradfordandsons (without the animation).
And I guess adding an icon might nicely demonstrate what you can do with the summary.

File: demo/src/index.css [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+details > summary {
+  cursor: pointer;
```
Comment (votemike) [inline demo/src/index.css:13]: Seems to keep a pointer in the details too 😕

Comment (markwylde) [reply demo/src/index.css:13]: Fixed in 81fd52b2816f32d88159044f88314bcaaf74ef85.

**Review by votemike — APPROVED** [review]

**Pull request merged**